netlify.toml

```diff
@@ -7,3 +7,8 @@
     X-Frame-Options = "deny"
     Content-Security-Policy = "default-src 'self'; script-src 'self' blob: https://assets.dcube.cloud https://*.wogaa.sg https://assets.adobedtm.com https://www.google-analytics.com https://cdnjs.cloudflare.com https://va.ecitizen.gov.sg https://*.cloudfront.net https://printjs-4de6.kxcdn.com https://unpkg.com https://wogadobeanalytics.sc.omtrdc.net 'unsafe-eval'; object-src 'self'; style-src 'self' https://fonts.googleapis.com/ https://*.cloudfront.net https://va.ecitizen.gov.sg https://*.wogaa.sg https://cdnjs.cloudflare.com https://datagovsg.github.io 'unsafe-inline'; img-src *; media-src *; frame-src https://wogaa.demdex.net/ https://*.youtube.com https://*.youtube-nocookie.com https://*.vimeo.com; frame-ancestors 'none'; font-src * data:; connect-src 'self' https://dpm.demdex.net https://www.google-analytics.com https://stats.g.doubleclick.net https://*.wogaa.sg https://va.ecitizen.gov.sg https://ifaqs.flexanswer.com https://*.cloudfront.net https://fonts.googleapis.com https://cdnjs.cloudflare.com https://wogadobeanalytics.sc.omtrdc.net;"
 
+[[headers]]
+# Define which paths this specific [[headers]] block will cover.
+for = "/documents/*"
+  [headers.values]
+  Access-Control-Allow-Origin = "https://verify.gov.sg"
```

Comment:
just put * ? do we care ? or *.gov.sg ?

Reply:
its either * or exact, cannot be *.gov.sg

I think no need * unless we have a use case for multiple sites to load the json?